spec.md

```diff
@@ -372,6 +372,10 @@ The Location header will contain the registry URL to access the accepted layer f
 header returns the canonical digest of the uploaded blob which MAY differ from the provided digest. Most clients MAY
 ignore the value but if it is used, the client SHOULD verify the value against the uploaded blob data.
 
+The registry MAY treat the `from` parameter as optional.
+
+If MAY cross-mount the blob referred to by the `digest` even if the `from` paramer is omitted or the repository in passed in the `from` does not container the blob.
```

Comment:
```suggestion
It MAY cross-mount the blob referred to by the `mount` parameter even if the `from` parameter is omitted or the `from` repository does not contain the blob.
```

Reply:
Thanks.